### PR TITLE
HuggingFace pytorch-pretrained-bert BERT models

### PR DESCRIPTION
**Pull request by VictorSanh**
Hi,
Following this issue https://github.com/huggingface/pytorch-pretrained-BERT/pull/506, here's a PR to integrate BERT models from pytorch-pretrained-bert into torch hub.

Tell me if something seems wrong or if changes are needed.

I am not sure to fully understand whether I should fill in `layout`, `background-class`, `body-class`. I imagine they are automatic parameters for torch.hub landing. Can I also include HuggingFace logo somewhere? If so, how?

### CODE REVIEW

**Comment by ailzhang** [comment]
@VictorSanh Yea of course, feel free to put a log in `/images` and put the image name in `image` field where the it was `pytorch-logo.png`. 
I will update the template to reflect that :D

File: huggingface_pytorch-pretrained-bert_bert.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# Tokenized input
+text = "[CLS] Who was Jim Henson ? [SEP] Jim Henson was a puppeteer [SEP]"
+tokenized_text = tokenizer.tokenize(text)
+indexed_tokens = tokenizer.convert_tokens_to_ids(tokenized_text)
+```
+
+We can get the hidden states computed by `bertModel`.
+
+```python
+model = torch.hub.load('huggingface/pytorch-pretrained-BERT', 'bertModel', 'bert-base-cased', force_reload=False)
```
Comment (ailzhang) [inline huggingface_pytorch-pretrained-bert_bert.md]: Also if you could do `import torch` here and in the code block below to make it a standalone runnable script, it would be great. 
We're thinking about setting up a CI to make sure thing runs fine (instead of manual check ;) . So this will help us a lot.

Comment (VictorSanh) [reply huggingface_pytorch-pretrained-bert_bert.md]: Updated for standalone snippets.
Thanks for your inputs!

File: huggingface_pytorch-pretrained-bert_bert.md [hunk 2]
```diff
@@ -59,22 +62,31 @@ segments_ids = [0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1]
...
     encoded_layers, _ = model(tokens_tensor, segments_tensors)
 ```
 
 We can predict masked tokens using `bertForMaskedLM`.
 
 ```python
-model = torch.hub.load('huggingface/pytorch-pretrained-BERT', 'bertForMaskedLM', 'bert-base-cased', force_reload=False)
-model.eval()
+import torch
+tokenizer = torch.hub.load('huggingface/pytorch-pretrained-BERT', 'bertTokenizer', 'bert-base-cased', do_basic_tokenize=False, force_reload=False)
```
Comment (soumith) [inline huggingface_pytorch-pretrained-bert_bert.md]: i think the `force_reload=False` can be removed from all of the calls right...

Comment (VictorSanh) [reply huggingface_pytorch-pretrained-bert_bert.md]: Done in cb1747b5f58935b77a1d779b2ce16190ea483d1e !
